vivisect/parsers/elf.py

```diff
@@ -490,7 +477,7 @@ def loadElfIntoWorkspace(vw, elf, filename=None, baseaddr=None):
             logger.debug("DYNAMIC:\t%r", d)
         
         dval = d.d_value
-        if d.d_tag in dt_rebase and addbase:
+        if d.d_tag in Elf.dt_rebase and addbase:
```

Comment:
It bothers the tidiness portion of my brain that you declare dt_rebase in the vstruct constants, but then pull it from the main `Elf` import.

Reply:
it bothers the tidiness of my brain that they're in vstruct(although i completely understand why we chose to put them there).  they're still in Elf to make access more intuitively pleasing (and easy).

i understand what you're saying, but i still believe I've chosen the better option

Reply:
No I mean if you imported the vstruct module to pull dt_rebase that would assuage my feelings of untidiness. Because I would like to get away from the "from foo import *" pattern since makes it more difficult to track what's in the current namespace.

Reply:
ahhh.  but i'd prefer to keep pulling from Elf, not vstruct (yes, i know i'm actually pulling from vstruct, but that is *way* not intuitively pleasing from a coding perspective).

Reply:
i think it's about time to rename `Elf.elf_lookup` to something more like `Elf.const`